macs/nix-darwin.nix

```diff
@@ -37,6 +37,12 @@ in
     "extra-experimental-features" = [ "nix-command" "flakes" ];
     max-jobs = 4;
     cores = 2;
+    "system-features" = [
+      "apple-virt"
+      "nixos-test"
+      "big-parallel"
+      "nixos-test"
```

Comment:
One `"nixos-test"` should suffice 😄

Reply:
I don't know what you're talking about, there's definitely only one.... (nice catch, thanks, derp)